### PR TITLE
Fixes #505 Added new svg artwork

### LINKED ISSUE #505: Update the SUSI svg with the new one
**Actual Behaviour**

Recently orbitter suggested a new svg for SUSI. The link is here : https://github.com/fossasia/susi_artwork/blob/master/logo/svg/SUSI.AI_Letters_Blue_Large.svg

**Expected Behaviour**

We should update our svg file with new one

**Would you like to work on the issue?**
@uday96 @rishiraj824 @isuruAb @mariobehling Please give your reviews whether this should be done or not. If yes, then i would like to solve this issue

### PR DESCRIPTION
**Pull request by madhavrathi**
Fixes issue #505

Changes: Added a new SVG logo for SUSI.AI. This is not a priority but the issue was opened 12 days ago. We should try to minimize issues in the tracker.

Demo Link: http://susi-newsvg.surge.sh/

Screenshots for the change: 
hehe

@rishiraj824 @uday96 @isuruAb Please Review
Thanks

### CODE REVIEW

**Comment by madhavrathi** [comment]
@rishiraj824 Good point. I will update the PR accordingly. 
Thanks

**Comment by madhavrathi** [comment]
@rishiraj824 @uday96 @isuruAb Please review http://susi-newsvg.surge.sh/
Thanks

File: src/components/ChatApp/ChatApp.css [hunk 1]
```diff
@@ -21,7 +21,7 @@ filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#183850', end
   overflow: hidden;
 }
 .app-bar{
-  background-image: url('../../images/susi-white.svg');
+  background-image: url('../../images/susi-logo-white.png');
```
Comment (uday96) [inline src/components/ChatApp/ChatApp.css]: Please increase the size of the logo. The height is smaller compared to the present logo

Comment (madhavrathi) [inline src/components/ChatApp/ChatApp.css]: The text vs height ratio seems good to me. on increasing height the size of whole app bar will change. @rishiraj824 @isuruAb Please provide your inputs here.
Thanks

**Comment by madhavrathi** [comment]
@rishiraj824 Please review. I have reduced the height.

**Comment by mariobehling** [comment]
@madhavrathi Thanks for the improvement. However, this is not a feature improvement or urgent bug and therefore is low priority. I am merging this, but I wonder why you are not taking up some real coding issues with high and urgent priority. Please help to make this project work now and provide real project improvements.

**Comment by madhavrathi** [comment]
@mariobehling I will work on priority issue from now on. I was looking to get started with this https://github.com/fossasia/chat.susi.ai/issues/586 since the assigned member has shown no progress can I work on this. This is the only thing left in the top bar of our web chat project. After this, i will work on susi skill cms project as there are many priority issues there.
Thanks

**Pull request merged**